.github/actions/set-up-walrus/action.yaml

```diff
@@ -65,8 +65,10 @@ runs:
         INPUT_SUI_NETWORK: ${{ inputs.SUI_NETWORK }}
         INPUT_WALRUS_CONFIG: ${{ inputs.WALRUS_CONFIG }}
       run: |
+        curl https://storage.googleapis.com/mysten-walrus-binaries/walrus-${{ inputs.SUI_NETWORK }}-latest-ubuntu-x86_64 -o bin/walrus
```

Comment:
This line appears to be redundant with the next line, but slightly different. Maybe a typo?

Reply:
What is the change here attempting to address?